client/src/services/ServiceWorker/MessageChannel.js

```diff
@@ -43,6 +44,12 @@ export default class MessageChannel {
 					};
 				})
 				.catch((error) => {
+					Sentry.withScope(function (scope) {
+						scope.setTag('messageChannel', 'true');
```

Comment:
We should be careful when track error on SW alot, I suggest to catch error on UI side first.